layer/projects/ray_workflow_project_runner.py

```diff
@@ -63,6 +74,13 @@ def run(self) -> Run:
         ray.shutdown()
         return run
 
+    def _get_ray_proxy_url(self) -> str:
+        from urllib.parse import urlsplit, urlunsplit
+
+        url = list(urlsplit(str(self._config.url)))
+        url[1] = f"ray.{url[1]}"
+        return urlunsplit(url)
```

Comment:
I don't this is what we need there, but I'm adding a `rayGatewayDomain` to `/__details.json` which we can just fetch the same way we fetch the auth0 URL.